examples/port-2-alibaba2-connection/main.tf

```diff
@@ -0,0 +1,65 @@
+provider "equinix" {
+  client_id     = var.equinix_client_id
+  client_secret = var.equinix_client_secret
+}
+
+provider "alicloud" {
+  access_key = var.access_key
+  secret_key = var.secret_key
+  region = var.region
+}
+
+resource "random_string" "random" {
```

Comment:
This should be in the documentation along with `connection_name       = "${var.connection_name}${random_string.random.result}"` to show the customer it's an option but we aren't forcing them to do it.

Reply:
Addressed with commit: [e2c6a1aff16a60d8cd9bdf5a6c1d4a2e5eb755cb](https://github.com/equinix/terraform-equinix-fabric/pull/155/commits/e2c6a1aff16a60d8cd9bdf5a6c1d4a2e5eb755cb)